src/api.js

```diff
@@ -13,39 +13,20 @@ if (!hfpUrl) {
   console.error("HFP GraphQL URL not set!");
 }
 
+const joreCache = new InMemoryCache();
+
 const joreClient = new ApolloClient({
   link: new HttpLink({uri: joreUrl}),
-  cache: new InMemoryCache({
-    dataIdFromObject: (obj) => {
-      if (obj.__typename === "Stop") {
-        return `Stop:${obj.stopId}`;
-      }
-
-      if (typeof obj.nodeId !== "undefined") {
-        return obj.nodeId;
-      }
-
-      if (obj.__typename === "Line") {
-        return `${obj.lineId}:${obj.dateBegin}:${obj.dateEnd}`;
-      }
-
-      return defaultDataIdFromObject(obj);
-    },
-  }),
```

Comment:
The custom `dataIdFromObject` function resulted in some very hard and headache-y to debug issues so I found it best to let Apollo handle it.